spark/src/test/scala/ai/chronon/spark/test/FetcherTest.scala

```diff
@@ -74,7 +74,10 @@ class FetcherTest extends TestCase {
     val singleFileMetadataStore = new MetadataStore(inMemoryKvStore, singleFileDataSet, timeoutMillis = 10000)
     inMemoryKvStore.create(singleFileDataSet)
     // set the working directory to /chronon instead of $MODULE_DIR in configuration if Intellij fails testing
-    val singleFilePut = singleFileMetadataStore.putConf(confResource.getPath)
+    val acceptedEndPoints = List(MetadataEndPoint.ConfByKeyEndPointName)
```

Comment:
I think we should break it out into its own test. There is a metadataexportertest.scala file

Reply:
Hi @nikhilsimha I need to update the tests here because the original putConf method has been removed from metadataStore file.